public/js/remotelyHostedDatasets.js

```diff
@@ -25,28 +25,57 @@ $(document).ready(function(){
             url: Routing.generate("pelagos_app_ui_remotelyhosteddatasets_post", {udi : $("#udiInput").val().trim()}),
         }).done(function(data, textStatus, jqXHR){
             var messageType = "success";
+            var modalBool = false;
             //return informative message for 202 code
             if (202 === jqXHR.status) {
+                console.log(data);
                 messageType = "warning";
+                if (data.includes("CONFIRM:")) {
+                    data = data.replace("CONFIRM:","");
+                    var confirmBool = true;
+                }
             } else {
                 //reset table
                 $("#remotelyHostedDatasetsTable").pelagosDataTable();
             }
-            var n = noty(
```

Comment:
Understand you are passing JSON, these are 90% the same, so you can make a base json, then add or remove properties based on the condition.